cube/src/main/java/zone/cogni/asquare/cube/convertor/data2shacl/ShaclGenerator.java

```diff
@@ -460,12 +493,13 @@ private boolean askQuery(RdfStoreService rdfStoreService, String queryFile, Map<
 
   private @Nonnull
   List<String> getProperties(@Nonnull RdfStoreService rdfStoreService,
-                             @Nonnull Resource targetClass) {
+                             @Nonnull Resource targetClass,
+                             @Nonnull Configuration configuration) {
```

Comment:
parameter order: go from most "stable" to most changeable. I would move configuration to the first spot, link in other methods.